src/api/index.js

```diff
@@ -18,17 +20,21 @@ module.exports = {
         data: JSON.stringify(item)
       },
       function(error, data) {
-        if (error) callback(error);
+        if (error) {
+          return callback(error);
+        }
         let status = JSON.parse(data).status;
         callback(null, status);
       }
     );
   },
-  post: function postItem(item, callback) {
+  create: function postItem(item, callback) {
```

Comment:
Weird mix of 'update' and 'post', 'create' is more inline with 'update' and is nicely semantic.